grpc/sa-wrappers.go

```diff
@@ -283,6 +283,26 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	if req == nil {
```

Comment:
As with the wrappers for `AddCertificate` this and below should also check that the fields supplied are not `nil` (or it should be done in the SA method instead) as currently they are also being blindly deref'd.

Reply:
Addressed in the replies to CPU's comments above.